AggregateMetrics/AggregateMetrics/AzureWebApp/SumUpGauge.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    using System.Collections.Generic;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two;
+
+    /// <summary>
+    /// Gauge that sums up the values of different gauges.
+    /// </summary>
+    public class SumUpGauge : ICounterValue
+    {
+        private string name;
+
+        private List<FlexiblePerformanceCounterGauge> gaugesToSum;
```

Comment:
for a good practice - mark this collection readonly so you'll never write a code that will attempt to change it